### PR TITLE
Bump the version for the next release

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Comment by Wovchena** [comment]
@vladimir-dudnik, ping

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Review by vladimir-dudnik — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # [OpenVINO™ Toolkit](https://01.org/openvinotoolkit) - Open Model Zoo repository
-[![Stable release](https://img.shields.io/badge/version-2021.4-green.svg)](https://github.com/openvinotoolkit/open_model_zoo/releases/tag/2021.4)
+[![Stable release](https://img.shields.io/badge/version-2021.4-green.svg)](https://github.com/openvinotoolkit/open_model_zoo/releases/tag/2022.1)
```
Comment (alalek) [inline README.md:2]: Still 2021.4 in `version-2021.4-green.svg`.

Also tagged releases has mismatched links: https://github.com/openvinotoolkit/open_model_zoo/blame/2021.4.2/README.md#L2
Release process should be improved.

Comment (vladimir-dudnik) [reply README.md:2]: cc @Wovchena

**Pull request merged**